pkg/snapstore/snapstore_suite_test.go

```diff
@@ -41,7 +42,7 @@ var _ = BeforeSuite(func() {
 	logrus.Infof("Starting test server...")
 	th.SetupHTTP()
 	initializeMockSwiftServer(testObj)
-	swiftStore = snapstore.NewSwiftSnapstoreFromClient(bucket, prefix, "/tmp", 5, fake.ServiceClient())
+	swiftStore = snapstore.NewSwiftSnapstoreFromClient(bucket, prefixV2, "/tmp", 5, fake.ServiceClient())
```

Comment:
Can we add the cases for v1 only, v2 only as well as v1 and v2? Or is it just initialization and the cases are anyway covered later?

Reply:
It was already there. I removed it.